plugins/azure-sites/src/api/AzureSitesApi.ts

```diff
@@ -29,6 +30,6 @@ export const azureSiteApiRef = createApiRef<AzureSitesApi>({
 /** @public */
 export type AzureSitesApi = {
   list: (request: AzureSiteListRequest) => Promise<AzureSiteListResponse>;
-  start: (request: AzureSiteStartStopRequest) => Promise<void>;
-  stop: (request: AzureSiteStartStopRequest) => Promise<void>;
+  start: (request: AzureSiteStartStopRequest, entity: Entity) => Promise<void>;
+  stop: (request: AzureSiteStartStopRequest, entity: Entity) => Promise<void>;
```

Comment:
the idea with these single-object-parameter methods is that it's convenient to have it all in one place when evolving over time; the new entity parameter should have gone inside the `Azure*Request` type